build.gradle

```diff
@@ -1,46 +1,150 @@
 plugins {
-    id 'com.github.eerohele.dita-ot-gradle' version '0.4.2'
+    id 'com.github.eerohele.dita-ot-gradle' version '0.4.3'
+    id 'com.github.eerohele.saxon-gradle' version '0.3.0'
 }
 
-ditaOt.dir System.getProperty('dita.home', '../')
+import com.github.eerohele.DitaOtTask
+import com.github.eerohele.SaxonXsltTask
 
-import com.github.eerohele.AntBuilderAssistant
-def antBuilder = AntBuilderAssistant.getAntBuilder(project)
+def getPropertyOrDefault(String name, def defaultValue) {
+    hasProperty(name) ? findProperty(name) : defaultValue
+}
 
-task pdf {
-  doLast {
-    antBuilder.execute {
-      ant(antfile: 'build.xml', target: 'pdf')
+String ditaHome = getPropertyOrDefault('ditaHome', '../')
```

Comment:
Could this be something like

```
String ditaHome = getPropertyOrDefault('ditaHome', buildscript.sourceFile.getParentFile().getParentFile().getAbsolutePath())
```

This way the default we would not need to define the `ditaHome` property.

Reply:
Done.